content/using-sagemaker-managed-spot-training/notebooks/configuring_libraries.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Configuring Libraries for Managed Spot Training"
+chapter: false
+weight: 30
+---
+
+Multiple libraries and SDKs such as the AWS Python SDK (boto3) and the Amazon SageMaker Python SDK support executing training jobs with Spot Instances. This workshop provides example notebooks that leverage multiple popular libraries and SDKs so that when working through the workshop you can choose to gain experience with those most relevant to your workload, or you can execute all of them as time permits to become familiar with various alternatives.
+
+## Available Notebooks
+
+* sagemaker-built-in-deepar - Example notebook for training and hosting a DeepAR model with the SageMaker Python SDK.
```

Comment:
Perhaps adding links to the spot-labs notebooks may help for others to explore ?  (although the link to the folder is provided in the access_jupyterlab.md)